### PR TITLE
Animation is not available didn't use it

### PR DESCRIPTION
**Pull request by marcus7777**

### CODE REVIEW

File: kwc-blockly-toolbox.html [hunk 1]
```diff
@@ -289,7 +289,7 @@
                         duration = size.height / 3;
                         this.style.overflowY = 'hidden';
                         for (let i = 0; i < buttons.length; i++) {
-                            if (over) {
+                            if (over && buttons[i].animate) {
```
Comment (paulvarache) [inline kwc-blockly-toolbox.html]: Maybe a one time check with `const canAnimate = 'animate' in HTMLElement.prototype` and then use the var

**Comment by paulmck21** [comment]
Is it possible to check the browser and apply a check on which browsers we try to do animations on? This would mean not having to come up with a way to check for each animatable element.

**Comment by paulvarache** [comment]
@paulmck21 Better to check the feature than the browser

**Comment by paulmck21** [comment]
@paulvarache true, guess I meant a way to check that the browser supports the feature

**Comment by marcus7777** [comment]
like this? @paulvarache

**Pull request merged**